src/PrestaShopBundle/EventListener/MultishopCommandListener.php

```diff
@@ -31,6 +31,9 @@
 use Symfony\Component\Console\Exception\LogicException;
 use Symfony\Component\Console\Input\InputOption;
 
+/**
+ * @deprecated since 1.7.7, to be removed in the next major version.
```

Comment:
can be kept, but I'm wondering if someone is using it as it's not documented

Reply:
Must be kept then 😢